source/summary.rst

```diff
@@ -132,30 +176,36 @@ Supplementary key consistency through ClaimChains
 
 We discuss a variant of ClaimChain_, a distributed key consistency scheme,
 in which all cryptographic checks are performed on the end-point side.
-ClaimChains are self-authenticated hash chains whose blocks contains statements
-about key material of the ClaimChain owner and her contacts.
+ClaimChains are self-authenticated hash chains whose blocks contain statements
+about key material of the ClaimChain owner and the key material of her contacts.
 The "head" of the ClaimChain, the latest block,
 represents a commitment to the current state,
 and the full history of past states.
 
-ClaimChain data structures tracks all claims about public keys
-and allows other peers to automatically verify the integrity of claims.
+ClaimChain data structures track all claims about public keys
+and enable other peers to automatically verify the integrity of claims.
 ClaimChains include cryptographic mechanisms
 to ensure the *privacy of the claim it stores*
 and the *privacy of the user's social graph*.
-This means that only authorized users can access the key material and
-cross-references being distributed. In other words, neither providers
+Only authorized users can access the key material and
+the cross-references being distributed. In other words, neither providers
 nor unauthorized users can learn anything about the key material
 in the ClaimChain and the social graph of users
 by just observing the data structure.
 
-ClaimChain also prevents users (or a network adversary who impersonates users)
-from *equivocating* to other users about their cross-references.
-That is, Alice should *not* be able to show different versions
-of a cross-reference of Bob's key to different users, i.e.,
-she cannot show one version only to Carol and only the other to Donald.
-If such equivocation were possible, it would hinder the ability to
+Private claims could be used by malicious users (or a network adversary who
+impersonates users) to *equivocate*, i.e.,
+present a different view of they keys they have seen to their peers.
+For example,
+Alice could try to equivocate by showing different versions of a cross-reference
+of Bob's key to Carol and Donald.
+Such equivocations would hinder the ability to
 resolve correct public keys.
+Therefore, ClaimChain prevents users (or a network adversaries)
+from *equivocating* to other users about their cross-references.
+
+..
+  TODO: why the details about Autocrypt headers and claimchain integration here?
```

Comment:
feel free to strike what you feel is redundant here. i agree there is redundancy and it's good to strike it in the summary.